ubg/src/mafia/MafiaDay.js

```diff
@@ -73,14 +75,17 @@ function MafiaDay({mafiaKill, doctorSave, usersData,
    */
   function startDay() {
     endGame();
-    if (!end) {
+    if (!end && !initialize) {
+      const allPlayers = [];
       if (mafiaKill && mafiaKill.uid !== doctorSave.uid) {
         usersCollection.doc(mafiaKill.uid).update({alive: false});
         setDeathText(mafiaKill.displayName + ' was killed last night');
+        room.update({
+          aliveCount: aliveCount - 1,
```

Comment:
I am not sure, but I think that this will decrease aliveCount for each player, because this code runs on the browser of all players.

Reply:
Maybe you could just keep this in the redux state.